lib/ui/view/sign_up/sign_up_viewmodel.dart

```diff
@@ -38,6 +42,8 @@ class SignUpViewModel extends FormViewModel {
   void navigateToHome() => navigator.navigateTo(Routes.navBarView);
   void navigateToSignIn() => navigation.navigateTo(Routes.loginView);
   void navigateToOTPView() => navigation.navigateTo(Routes.oTPView);
+  // void navigateToOrganizationView(GoogleSignInAccount user) =>
```

Comment:
Please remove code that is not in use

Reply:
Done.
The conflicting file "organisational view" is perfect now
I had a poor connection that's why it's throwing the dip